8.2.10 Release

su-sws-jira opened the pull request:
# DO NOT DELETE

Commit:
feat(STN-368):  tighten spacing for traditional theme cards (#600)

* feat(STN-368): pairing with design on tighter editorial look for cards
* feat(STN-368): editorial changes to date horizontal structured card
* refactor(STN-368): adds xxxs spacing unit
* fix(STN-368): too many tradl mixins nested
* feat(STN-368): adds compiled CSS

kaseybon committed:
feat: utility classes to affect the width of card component images (#601)

STN-362

Commit:
feat(STN-342) : Blue Jay Traditional Color Pairing (#598)

* feat: update Sand Color Pairing to the Cardinal

* feat: add / update Traditional color pairing architecture to include Bluejay

* feat: update Traditional Cardinal color pairings to match spreadsheet

* feat: update traditional well styles

* feat: update secondary nav to use traditional color pairing values

* feat: update main nav dropdown item hover state to have traditional values

* feat: update traditional navigation dropdown color

* feat: updating of icons and links for the traditional color pairing

* feat: update dark footer font awesome values for traditional color pairings

* feat: update local footer dark variant secondary buttons for traditional themnes

* feat: update local footer dark variant table header for traditional themes

* feat: update vertical linked card styles for the traditional theme

* feat: add Sass test for Blue Jay color pairing

* feat: clean up commented code

* feat: implement review feedback

* feat: add additional ways of targeting fontawesome hover states

* feat: add compiled CSS

kaseybon committed:
STN-377: Utility Classes for Table Column Widths (#605)

* feat: add utility classes to adjust the column widths in table views
* chore: add compiled css

Co-authored-by: Bryan Braun <bryan@heysparkbox.com>

su-sws-jira committed:
8.2.10

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

su-sws-jira committed:
Updated dependencies May 20 2020

Commit:
feat(STN-372 ): Theme Animations (#599)

* feat: refactor the transition function to check for a theme and output the proper transition values set for that theme

* feat: update instances of $hb-animation- variables to use the hb-transition function

- find all transitions and notate with a TODO flag so they can be reviewed with a designer

* feat: implement design feedback on traditional transitions

* feat: add image animations to linked images in the WYSIWYG

* feat: add Sass tests for the hb-transition function

* feat: clean-up / remove extra comments

* feat: implement review feedback

* feat: update animation function variables

* feat: add compiled CSS

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

pookmish committed:
Merge branch 'develop' into 8.2.10-release

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

bryanbraun committed:
feat(STN-376): add indicators to unpublished pages and menu links (#603)

* feat(STN-376): add indicators to unpublished pages and menu links
* commit css

pookmish committed:
HSD8-808 Provide config page for google analytics and disabling rabbit hole

pookmish committed:
HSD8-806 added some styles and tags to the minimal html

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

bryanbraun committed:
fix(STN-374): Row paragraph style field permissions (#610)

* feat: set new default field permissions for row styles

This ensures "site-managers" and "contributors" can see and update the
row styles field.

These permissions changes should only apply to newly provisioned sites
because permissions are currently config-ignored, as shown in
/config/default/config_ignore.settings.yml.

* test: update behat tests to include new default permissions

* feat: add 'View anyones value' for anonymous and authenticated users

Commit:
feat(STN-391): adds styles for the raised card utility (#611)

* feat(STN-391): adds styles for padded card utility
* feat: rename padded-cards to raised-cards
* chore: commit compiled css

Co-authored-by: Bryan Braun <bryan@heysparkbox.com>

Commit:
feat(STN-297): equal grid cards (#609)

* wip(STN-297): first attempt without JS

* feat(STN-297): adds JS for equal height grid cards

* fix(STN-297): IE11 bug

* refactor: rename the utility class that stretches the vertical linked cards and equalizes the height of the title area

STN-297

* chore: commit compiled css

Co-authored-by: Kasey Bonifacio <kasey@heysparkbox.com>

pookmish committed:
HSD8-809 Added ckeditor_liststyle module

pookmish committed:
HSD8-811 HSD8-813 Changed private page label and exclude private files from row

pookmish committed:
HSD8-814 Default images for events and publications

pookmish committed:
HSD8-812 Allow permissions for the hero overlay color field

Commit:
feat(STN-351): QA Feedback 16 (#608)

* feat: update background-color of colorful themed pills to improve color contrast / accessibility

* feat: prevent images on node details pages from shrinking

* feat: improve color contrast / accessibility of more links for the local footer dark variant

* feat: add margin top to cards within row wells when a heading is not present

* feat: add styles to allow accordions to span the width of rows within full and limited width wells

* feat: adjust spacing around heros and carousels when placed in a row with a well

* feat: adjust styles to center more link traditional arrow

* feat: write styles to prevent overlapping numbers and bullets in nested list elements

* feat: implement solution for icons created for the date stacked horizontal card to render as expected all the time

- We are using classes created by Drupal for the styling of the icons. Sometimes these classes auto-generate a modifer to the end of the class name. In an effort to cover all bases, use regex to render styles as long as the core class name matches.

* feat: add styles that allow for content in the main top text area of node details pages to break apart and flow alongside images

* feat: ensure that hero and carousels within rows expand their full width

* feat: add spacing to accordion's to prevent summary content from overlaying the plus icon

* feat: add compiled CSS

pookmish committed:
Removed patch on bricks module

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

pookmish committed:
fixup! Removed patch on bricks module

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

pookmish committed:
locked paragraphs to 1.11 to help with field groups

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

Commit:
feat: update behat tests for contributor and site manager

su-sws-jira committed:
Updated dependencies May 27 2020

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

pookmish committed:
Sparkbox Sprint 17

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

pookmish committed:
added BLT task that will stage all the databases from production to test

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

pookmish committed:
Adjusted ui patterns patch to fix missing link to content fields

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr606-622ahfxkt0d2tgtriky31ipafyho4vh7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599

Visual Diffs:
- https://dashboard.tugboat.qa/5ec457b1c3fb2439a389e599/visualdiffs

silverli commented on patches/contrib/ui_patterns_views-preview.patch:
```diff
@@ -96,7 +96,7 @@ index 388b287..6f835d4 100644
 +    // render this even if set to exclude so it can be used elsewhere.
 +    $field_output = $view->style_plugin->getField($row->index, $id);
-+    $empty = $field->isValueEmpty($field->getValue($row), $field->options['empty_zero']);
++    $empty = $field->getPluginId() == 'field' && $field->isValueEmpty($field->getValue($row), $field->options['empty_zero']);
```
👍

silverli replied:
this works to fix an issue I was seeing when we mapped "Link to Content" to other fields in a pattern view.

previously, the field for link to content would always render as null, even when there was a node detail page to link to.

<img width="851" alt="Screen Shot 2020-06-01 at 5 56 06 PM" src="https://user-images.githubusercontent.com/12848168/83463099-0446dc80-a43b-11ea-9197-d0fd75bb4e57.png">

pookmish committed:
Adjusted line heights for vertical and horizontal cards
